frontend/src/store/users.js

```diff
@@ -52,7 +52,7 @@ export default function usersReducer (state = {}, action) {
 
     switch(action.type) {
         case RECEIVE_USER:
-            return {...newState, [action.user._id]: {...action.user, itineraries: action.itinerariesIds, likedItineraries: action.likedItineraries} };
+            return {...newState, [action.user?._id]: {...action.user, itineraries: action.itinerariesIds, likedItineraries: action.likedItineraries} };
```

Comment:
this is not really right, incoming user will be set under 'undefined' key in store then